cmd/kubeadm/app/cmd/init.go

```diff
@@ -103,6 +103,13 @@ var (
 		If you are on a systemd-powered system, you can try to troubleshoot the error with the following commands:
 			- 'systemctl status kubelet'
 			- 'journalctl -xeu kubelet'
+		
+		Additionally a service may not have come up in docker. If that's the case, you can enumerate all docker
```

Comment:
How about changing `service` to `control plane component` ?

Reply:
Done